addons/hr_timesheet/views/res_config_settings_views.xml

```diff
@@ -27,7 +27,7 @@
                     </block>
                     <div name="section_leaves">
                         <block title="Time Off" name="timesheet_control">
-                            <setting company_dependent="1" documentation="/applications/services/timesheets/overview/time_off.html" help="Generate timesheets upon time off validation" id="timesheet_off_validation_setting">
+                            <setting company_dependent="1" documentation="/applications/services/timesheets/overview/time_off.html" help="Generate timesheets for validated time offs and public holidays" id="timesheet_off_validation_setting">
```

Comment:
```suggestion
                            <setting company_dependent="1" documentation="/applications/services/timesheets/overview/time_off.html" help="Generate timesheets for validated time off requests and public holidays" id="timesheet_off_validation_setting">
```
I spoke with Alexandra and she would prefer this small modification: "time offs" -> "time off requests"